joint_configs: [
  {id:  2, name: "r_shy_joint",   direction:  1, offset: -90},
  {id:  3, name: "head_ny_joint", direction:  1, offset: 0},
  {id:  4, name: "l_shy_joint",   direction: -1, offset: 90},
  {id:  5, name: "head_nz_joint", direction: -1, offset: 0},
  {id:  6, name: "r_hpz_joint",   direction:  1, offset: 0},
  {id:  7, name: "head_nx_joint", direction: -1, offset: 0},
  {id:  8, name: "l_hpz_joint",   direction:  1, offset: 0},
  {id:  9, name: "r_shx_joint",   direction:  1, offset: 0},
  {id: 10, name: "r_hpx_joint",   direction:  1, offset: 0},
  {id: 11, name: "l_shx_joint",   direction:  1, offset: 0},
  {id: 12, name: "l_hpx_joint",   direction:  1, offset: 0},
  {id: 13, name: "r_ely_joint",   direction:  1, offset: 0},
  {id: 14, name: "r_hpy_joint",   direction:  1, offset: 0},
  {id: 15, name: "l_ely_joint",   direction: -1, offset: 0},
  {id: 16, name: "l_hpy_joint",   direction: -1, offset: 0},
  {id: 17, name: "r_elx_joint",   direction:  1, offset: 0},
  {id: 18, name: "r_kny_joint",   direction: -1, offset: 0},
  {id: 19, name: "l_elx_joint",   direction:  1, offset: 0},
  {id: 20, name: "l_kny_joint",   direction:  1, offset: 0},
  {id: 21, name: "r_wry_joint",   direction:  1, offset: 90},
  {id: 22, name: "r_aky_joint",   direction:  1, offset: 0},
  {id: 23, name: "l_wry_joint",   direction: -1, offset: -90},
  {id: 24, name: "l_aky_joint",   direction: -1, offset: 0},
  {id: 26, name: "r_akx_joint",   direction:  1, offset: 0},
  {id: 28, name: "l_akx_joint",   direction:  1, offset: 0},
]